### PR TITLE
Bugfix/#42 add event status filter back in

### LINKED ISSUE #42: Tarbela keeps fetching and dispatching already sent events
During the implementation of #34, the event filtering by status NEW was lost. As a result, tarbela keeps fetching and dispatching events that it has already sent to Nakadi, over and over.

### PR DESCRIPTION
**Pull request by Grillkohle**
During #34 the query parameter addition for status = NEW was lost, this pull request adds it back in.

Fixes #42.

### CODE REVIEW

File: src/main/java/org/zalando/tarbela/config/ProducerConfiguration.java [hunk 1]
```diff
@@ -65,4 +76,15 @@ private AccessTokens initializeAccessTokens() {
...
                 producerConfigurations.getTokens().getAccessTokenUri()).getAccessTokens();
     }
+
+    //TODO: figure out how to test this
+    private URI addEventFilterQueryParameter(final URI uri) {
+        try {
+            return new URIBuilder(uri).addParameter(QUERY_EVENT_STATUS_FILTER_NAME, QUERY_EVENT_STATUS_FILTER_VALUE)
+                                      .build();
+        } catch (URISyntaxException e) {
+            log.error("Failed to launch producer with event URI: " + uri);
```
Comment (goekhanak) [inline src/main/java/org/zalando/tarbela/config/ProducerConfiguration.java]: maybe ```Failed to construct producer URI from:  ```

**Comment by ePaul** [comment]
:+1:

**Comment by Grillkohle** [comment]
👍

**Comment by ePaul** [comment]
:+1:

**Comment by goekhanak** [comment]
:+1:

**Pull request merged**